### PR TITLE
add initial instructions on HTML variables

### PR DESCRIPTION
**Pull request by mpadge**
@maurolepore Can you please check the Spanish version of this PR, and make any changes you think might improve it. Thanks!

### CODE REVIEW

**Review by maelle — APPROVED** [review]
Thanks a lot!

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ name: Enviar software para revisión
 about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
 ---
 
+<!---
+En la sección que sigue a continuación, introduzca los dos valores de (1) el mango de GitHub del autor del paquete y (2) la URL del repositorio. También se pueden especificar valores para autores de paquetes adicionales. Los valores para cada uno deben colocarse entre las dos secciones HTML, como se ilustra para el valor "author1". Los símbolos HTML que definen cada variable deben dejarse como están - así que el autor que envía se especifica sustituyendo sólo "@github_handle", mientras que se dejan los símbolos HTML adjuntos como están. Este comentario puede ser eliminado una vez que haya sido leído y comprendido.
```
Comment (maurolepore) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:7]: LGTM except for one small thing: The literal translation of "handle" to "mango" makes no sense to me. Instead I suggest "usuario" (for user). Also I suggest replacing the masculine "del autor" with "la persona que envio el paquete" or the feminine "la autora".

```
... (1) el [-mango-]{+usuario+} de GitHub [-del autor del-]{+de la persona que envio el+} paquete
```

Same here

```
así que [-el autor-]{+la persona+} que envía se especifica sustituyendo sólo "@github_handle"
```

**Review by maurolepore — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md (hunk 1 above)
Comment (mpadge) [reply .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md]: Thanks @maurolepore, amended as suggested.

File: .github/ISSUE_TEMPLATE/A-submit-software-for-review.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ about: Use this template to submit software for review
 
 ---
 
+<!---
+In the section that immediately follows, please enter the 2 values of (1) submitting author GitHub handle; and (2) Repository URL. Values for additional package authors may also be specified. Values for each must be placed between the two HTML sections, as illustrated for the "author1" value. The HTML symbols defining each variable must be left as they are - so the submitting author is specified by replacing "@github_handle" only, while leaving the enclosing HTML symbols as they are. This comment may be deleted once it has been read and understood.
```
Comment (noamross) [inline .github/ISSUE_TEMPLATE/A-submit-software-for-review.md:8]:
```suggestion
Below, please enter the values of (1) submitting author GitHub handle (replacing "@github_handle"); and (2) Repository URL (replacing "http:://repourl"). Values for additional package authors may also be specified replacing "@github_handle1," "@github_handle2" - delete these if not needed. DO NOT DELETE HTML symbols (e.g." <!--author1-->", "<!--repourl-->"). Replace "@github_handle" only.  This comment may be deleted once it has been read and understood.
```

An attempt at more clarity and brevity.   Please add `http://repourl` in the template - I don't think you have allowed me to edit the PR directly.

**Review by noamross — CHANGES_REQUESTED** [review]

**Comment by maurolepore** [comment]
Sorry, my bad. Where I suggested "envio" (submitted) I instead suggest "envía" (submists or submitting). Note the í not i.

**Comment by mpadge** [comment]
@maurolepore I requested a re-review of changes in response to suggestions by @noamross. Could you please give it another look-over? Thanks

**Review by maurolepore — APPROVED** [review]
LGTM thanks a lot!

**Comment by mpadge** [comment]
@noamross sorry about push access to my local branch - should be fixed now. Can you please approve? Thanks

File: .github/ISSUE_TEMPLATE/F-submit-statistical-software-for-review.md [hunk 3]
```diff
@@ -4,10 +4,13 @@ about: Use this template to submit statistical software for review
 
 ---
 
+<!---
+Below, please enter values for (1) submitting author GitHub handle (replacing "@github_handle@); and (2) Repository URL (replacing "https://repourl"). Values for additional package authors may also be specified, replacing "@github_handle1", "@github_handle2" - delete these if not needed. DO NOT DELETE HTML SYMBOLS. Replace only "@github_handle" and "https://repourl". This comment may be deleted once it has been read and understood.
```
Comment (noamross) [inline .github/ISSUE_TEMPLATE/F-submit-statistical-software-for-review.md:8]:
```suggestion
Below, please enter values for (1) submitting author GitHub handle (replacing "@github_handle@); and (2) Repository URL (replacing "https://repourl"). Values for additional package authors may also be specified, replacing "@github_handle1", "@github_handle2" - delete these if not needed. DO NOT DELETE HTML SYMBOLS (e.g. "<!--author1-->", "<!--end-author1-->). Replace only "@github_handle" and "https://repourl". This comment may be deleted once it has been read and understood.
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 4]
```diff
@@ -3,10 +3,13 @@ name: Enviar software para revisión
 about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
 ---
 
+<!---
+A continuación, introduzca los valores de (1) el usuario de GitHub de la persona que envía el paquete (sustituyendo "@github_handle") y (2) la URL del repositorio (sustituyendo "https://repourl"). También se pueden especificar valores para autores de paquetes adicionales, sustituyendo "@github_handle1", "@github_handle2" - elimine esto si no lo necesita. NO BORRAR LOS SÍMBOLOS HTML. Austituir sólo "@github_handle" y "https://repourl". Este comentario puede ser eliminado una vez que haya sido leído y comprendido.
```
Comment (noamross) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:7]:
```suggestion
A continuación, introduzca los valores de (1) el usuario de GitHub de la persona que envía el paquete (sustituyendo "@github_handle") y (2) la URL del repositorio (sustituyendo "https://repourl"). También se pueden especificar valores para autores de paquetes adicionales, sustituyendo "@github_handle1", "@github_handle2" - elimine esto si no lo necesita. NO BORRAR LOS SÍMBOLOS HTML (e.g., "<!--author1-->", "<!--end-author1-->"). Austituir sólo "@github_handle" y "https://repourl". Este comentario puede ser eliminado una vez que haya sido leído y comprendido.
```
Is "e.g.," correct here? I don't know if Latin abbreviations are expected in Spanish.

**Review by noamross — COMMENTED** [review]
I think we should keep these examples, on the assumption that people may not realize these are HTML comments.

**Comment by mpadge** [comment]
Thanks again @noamross - all updated (including "e.g." -> "por ejemplo" in Spanish template).

**Review by noamross — APPROVED** [review]

**Pull request merged**